Source/Extensions/Blazorise.DataGrid/_DataGridCellSelectEdit.razor.cs

```diff
@@ -26,4 +27,39 @@ public partial class _DataGridCellSelectEdit<TItem> : ComponentBase
     [Parameter] public EventCallback<object> CellValueChanged { get; set; }
 
     [Parameter] public bool ShowValidationFeedback { get; set; }
+
+    private System.Collections.Generic.List<(string text, object value, bool disabled)> selectItems;
```

Comment:
Better to have a designated model than the tupple.

Reply:
I'm fine with that. I will change after you take a look at approve the actual workaround.

Although to me I would be fine with value tuples in such a specific contained place where readability or navigation is unaffected in my opinion.